gsplat/cuda/csrc/forward.cu

```diff
@@ -269,15 +265,14 @@ __global__ void nd_rasterize_forward(
             }
 
             const float next_T = T * (1.f - alpha);
-            if (next_T <= 1e-4f) { // this pixel is done
-                // we want to render the last gaussian that contributes and note
-                // that here idx > range.x so we don't underflow
+            if (next_T <= 1e-4f) {
```

Comment:
why remove the comments?

Reply:
TBH I didn't really understand what the comment was saying so I removed it, but I stuck it back if you think it's helpful